examples/browser-to-server/package.json

```diff
@@ -5,14 +5,20 @@
   "type": "module",
   "scripts": {
     "start": "vite",
-    "go-libp2p-server": "cd ../go-libp2p-server && go build && ./go-libp2p-server"
+    "build": "vite build",
+    "go-libp2p-server": "cd ../go-libp2p-server && go build && ./go-libp2p-server",
+    "test": "npm run build && playwright test tests"
   },
   "dependencies": {
     "@chainsafe/libp2p-noise": "^11.0.0",
     "@multiformats/multiaddr": "^11.0.5",
-    "js-libp2p-webrtc": "../../",
     "it-pushable": "^3.1.0",
+    "js-libp2p-webrtc": "../../",
```

Comment:
actual package is now called @libp2p/webrtc, so we might as well call it here this as well.